tests/kafkatest/tests/templates/connect-file-sink.properties

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 name=local-file-sink
-connector.class=org.apache.kafka.connect.file.FileStreamSinkConnector
+connector.class=FileStreamSink
```

Comment:
Does the modification to these mean you've also run the Connect system tests with these changes and they pass?

Also, there are example files under `config/` with similar settings. Shall we update those as well?